core/tests/protractor/learnerDashboard.js

```diff
@@ -28,18 +28,23 @@ var ExplorationPlayerPage =
 var users = require('../protractor_utils/users.js');
 var LearnerDashboardPage =
   require('../protractor_utils/LearnerDashboardPage.js');
+var SubscriptionDashboardPage =
+  require('../protractor_utils/SubscriptionDashboardPage.js');
 
 describe('Learner dashboard functionality', function() {
   var creatorDashboardPage = null;
   var adminPage = null;
   var libraryPage = null;
   var learnerDashboardPage = null;
   var explorationPlayerPage = null;
+  var subscriptionDashboardPage = null;
 
   beforeEach(function() {
     creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
     libraryPage = new LibraryPage.LibraryPage();
     explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    subscriptionDashboardPage = new 
+    SubscriptionDashboardPage.SubscriptionDashboardPage();
```

Comment:
Indent continuation lines by 2 spaces. Do something like this:

    subscriptionDashboardPage = (
      new SubscriptionDashboardPage.SubscriptionDashboardPage());